src/navigation/image_database.cc

```diff
@@ -132,6 +237,9 @@ ImageDatabase::ImageDatabase(const std::tm *creationTime,
                              bool overwrite)
   : m_Path{ std::move(databasePath) }
 {
+    memset(&m_CreationTime, 0, sizeof(m_CreationTime));
```

Comment:
I think ``m_CreationTime{{0}}`` in the member initialiser list will achieve the same

Reply:
Good point!